db/seeds.rb

```diff
@@ -5,4 +5,19 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
\ No newline at end of file
+if Rails.env.development?
+  puts "Cleaning DB"
+  AdminUser.destroy_all
+  User.destroy_all
```

Comment:
La description de `db:seed` est "Load the seed data from db/seeds.rb".

Il y a d'autres taches dont le rôle est de remettre à zéro la base de données, comme `db:reset`.

Reply:
Le `db:reset` est connu pour avoir quelques failles, et ne fait qu'un `db:drop db:create db:migrate` donc impose de drop le base, donc réinitialiser les index etc.
Mais pour le coup çe ne change rien pour moi alors pas de souci pour le supprimer.

Reply:
C'est la description de Rails 3 que tu donnes là, mais tu es sur Rails 6. 

`db:reset` c'est 
> "Drops and recreates the database from db/schema.rb for the current environment and loads the seeds." 

(et c'est littéralement un `db:drop db:setup`).

et `db:setup` c'est 
> "Creates the database, loads the schema, and initializes with the seed data (use db:reset to also drop the database first)".

https://github.com/rails/rails/blob/6-0-stable/activerecord/lib/active_record/railties/databases.rake